unstract/connectors/src/unstract/connectors/databases/psycopg_handler.py

```
@@ -33,6 +40,33 @@ def execute_query(
+            # Handle race condition during concurrent CREATE TABLE IF NOT EXISTS.
+            # When multiple workers create the same table simultaneously,
+            # PostgreSQL may raise DuplicateTable even with IF NOT EXISTS clause
+            # due to race at the pg_type system catalog level.
+            if _is_create_table_if_not_exists(sql_query):
+                logger.info(
+                    f"Table '{table_name}' was created by concurrent process. "
+                    f"Continuing with existing table. (pg_type race condition)"
+                )
+                engine.rollback()
+                return
+            logger.error(f"DuplicateTable error: {e.pgerror}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)